### PR TITLE
Add description and demo

### PR DESCRIPTION
**Pull request by tencups**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
...
 
 [![Build Status](https://travis-ci.org/openmrs/openmrs-esm-patient-chart.svg?branch=master)](https://travis-ci.org/openmrs/openmrs-esm-patient-chart)
+
+
+## Overview
+This module allows developers to quickly and easily put together a 'chart' or 'flowsheet' style user interface for a patient, as opposed to an encounter-form-based UI.
+
+The flowsheet can have a bunch of tabs (representing physical pages) each having a different table. For example, there can be a history, physical condition, allergy, reactions and summary table.
+
+## Screenshots
```
Comment (suthagar23) [inline README.md]: Remove this headings

**Comment by suthagar23** [comment]
You need to improve your work on this by adding,
1. Developer documentation
2. Licenses
3. Wiki Pages

And add proper description to your PR

File: README.md [hunk 2]
```diff
@@ -1,3 +1,51 @@
...
+                     <property name="formId"><value>19</value></property>
+                 </bean>
+             </list>
+         </property>
+     </bean>
+ </property>
+</bean>
+```
+
+Link to Full [Wiki Page](https://wiki.openmrs.org/display/docs/Patient+Chart+Widgets+Module)
```
Comment (suthagar23) [inline README.md]: Add the related wiki pages inside a Wiki Documentation heading

File: README.md [hunk 3]
```diff
@@ -1,3 +1,51 @@
...
+License: CC BY 4.0
```
Comment (suthagar23) [inline README.md]: Add proper licensing inside the licensing header.

File: README.md [hunk 4]
```diff
@@ -1,3 +1,55 @@
...
+## Wiki Documentation
+- [OpenMRS Wiki - Patient Chart](https://wiki.openmrs.org/display/docs/Patient+Chart+Widgets+Module)
+- [OpenMRS Wiki - Patients](https://wiki.openmrs.org/display/docs/Patients)
+- [OpenMRS Wiki - Patient History](https://wiki.openmrs.org/display/docs/Patient+History)
+
+## License
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC%20BY%204.0-lightgrey.svg)](https://creativecommons.org/licenses/by/4.0/)
+
+
+
```
Comment (mavrk) [inline README.md]: Not Important : Remove extra whitespace

**Comment by suthagar23** [comment]
Hi @tencups , Please give some time to read our pull request requirements - https://wiki.openmrs.org/display/docs/Pull+Request+Tips. You need to add some changes to this pull request to get it merged.

**Comment by fatmali** [comment]
hi @tencups , I'm not sure this documentation was meant for this repo, this is not a Java module, it's a JavaScript single page application for the microfrontends project.

**Comment by jonathandick** [comment]
@tencups this looks like docs for a different module. This is a MF module as described by Fatma above. I'm going to close this ticket.

**Pull request closed without merging**